spring-cloud-stream-samples/pom.xml

```diff
@@ -21,6 +21,7 @@
 		<module>tap</module>
 		<module>double</module>
 		<module>extended</module>
+		<module>rxjava</module>
```

Comment:
Also, let's call it `rxjava-processor`. In the future (when we generalize) we might want to have sources and sinks with the same functionality.